pkg/apis/storage/validation/validation_test.go

```diff
@@ -2051,7 +2051,7 @@ func TestCSIDriverValidationUpdate(t *testing.T) {
 		{
 			name: "FSGroupPolicy invalidated",
 			modify: func(new *storage.CSIDriver) {
-				invalidFSGroupPolicy := storage.ReadWriteOnceWithFSTypeFSGroupPolicy
+				invalidFSGroupPolicy := storage.ReadWriteOnceWithFSTypeFSGroupPolicy //nolint:ineffassign
```

Comment:
@msau42 @jsafrane @mattcary Can you or someone storage-ish make a statement on the real fix here? it's clearly wrong...

Reply:
found a better way to create a invalidFSGroupPolicy